easybuild/easyconfigs/h/HDF5/HDF5-1.8.16-GCC-4.9.3-serial.eb

```diff
@@ -0,0 +1,22 @@
+name = 'HDF5'
+version = '1.8.16'
+versionsuffix = '-serial'
+
+homepage = 'http://www.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.3'}
```

Comment:
seems like you'll need to go with `gimkl` here to avoid this conflict:

```
Conflict found for dependencies of ParaView-4.4.0-gimkl-2.11.5-mpi: zlib-1.2.8-GCC-4.9.3 vs zlib-1.2.8-gimkl-2.11.5
```